src/bin/IntaRNA.cpp

```diff
@@ -96,7 +99,7 @@ int main(int argc, char **argv){
 		// parallelize this loop if possible; if not -> parallelize the query-loop
 		# pragma omp parallel for schedule(dynamic) num_threads( parameters.getThreads() ) shared(queryAcc,reportedInteractions,exceptionPtrDuringOmp,exceptionInfoDuringOmp) if(parallelizeTargetLoop)
 #endif
-		for ( size_t targetNumber = 0; targetNumber < parameters.getTargetSequences().size(); ++targetNumber )
+		for ( size_t targetNumber = target_range.from; targetNumber <= target_range.to; ++targetNumber )
```

Comment:
this disables multi-sequence support doesnt it?

Reply:
this only supports a single range so far... if you want to enable multiple ranges, you need an additional `for`-loop..